templates/3dsmax/scripts/PrepareVrayServices.bat

```diff
@@ -5,35 +5,10 @@ set VRAY_PORT=%3
 setx AZ_BATCH_ACCOUNT_URL %AZ_BATCH_ACCOUNT_URL% /M
 setx AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN %AZ_BATCH_SOFTWARE_ENTITLEMENT_TOKEN% /M
 
-IF "%VRAY_RENDERER%"=="VRayAdv" (
-
-    rem We need to create the config files first
-    echo.[Directories] > C:\Autodesk\3dsMax2018\vrayspawner.ini
-    echo.AppName=C:\Autodesk\3dsMax2018\3dsmaxio.exe >> C:\Autodesk\3dsMax2018\vrayspawner.ini
-    echo.[Directories] > C:\Autodesk\3dsMax2019\vrayspawner.ini
-    echo.AppName=C:\Autodesk\3dsMax2019\3dsmaxio.exe >> C:\Autodesk\3dsMax2019\vrayspawner.ini
-
-    IF "%MAX_VERSION%"=="2018" (
-	   start "vrayspawner2018" "C:\Autodesk\3dsMax2018\vrayspawner2018.exe" "-port=%VRAY_PORT%" > vrayexe.output.txt
-    ) 
-    IF "%MAX_VERSION%"=="2019" (
-	   start "vrayspawner2019" "C:\Autodesk\3dsMax2019\vrayspawner2019.exe" "-port=%VRAY_PORT%" > vrayexe.output.txt
-    )
-	IF "%MAX_VERSION%"=="2020" (
-	   start "vrayspawner2020" "C:\Autodesk\3dsMax2020\vrayspawner2020.exe" "-port=%VRAY_PORT%" > vrayexe.output.txt
+IF /I "%VRAY_RENDERER%"=="vray5" (
```

Comment:
`%VRAY_RENDERER%` comes from pool parameter `vrayRenderer` with value "VRay5", not "vray5"